app/assets/stylesheets/application.css

```diff
@@ -118,6 +118,7 @@ button,
 select,
 textarea {
 	color: #555;
+	background-color: white;
```

Comment:
This also applies to `input`, `button`, and `select`.  You should put this in the block right under this which is just for `textarea`.